rest/src/main/java/cz/incad/kramerius/rest/api/k5/client/pdf/PDFResource.java

```diff
@@ -393,7 +393,12 @@ public void write(OutputStream output)
                     LOGGER.log(Level.SEVERE, e.getMessage(), e);
                     throw new GenericApplicationException(e.getMessage());
                 } catch (OutOfRangeException e1) {
-                    LOGGER.log(Level.SEVERE, e1.getMessage(), e1);
+                    if (e1.getMessage().equals("too much pages")) {
```

Comment:
Díval jsem se do zdrojáků a OutOfRangeException se používá jenom při "too much pages", takže bych vyhodil ten if a vypisoval to jenom jako getMessage. Dále bych upravil hlášku na něco rozumnějšího, aby administrátor věděl co s tím má dělat. Např "too much pages for pdf generating - consider changing `config attrubute`.

Severitu nastav na warning, není to úplně běžný stav.